### ISSUE TITLE
No sound when set to automatic run

### ISSUE THREAD

**Issue by zarmedia1** [chundermike/rpi-fruitbox#42]
Fruitbox audio works fine when manually executing program in terminal.  Lines added to bashrc per user guide to auto run, program boots successfully (Pi set to boot to terminal with auto password fill-in), visually looks fine and plays, but no sound.

•audio output of pi configured to headphone out
•amixer set to output to headphone

Again, everything works as it should when manually started.  When auto-booting, no audio.  I want to auto-boot since this will be a standalone device.

I'm somewhat of a novice, so it's safe to assume it could be something low level/obvious that I'm overlooking.  Thanks for the assistance.

**Comment by Comet-boy** [comment]
Did you find a solution? I've got a similar issue!

**Comment by zarmedia1** [comment]
I don't think so.  Frustrating when developers won't respond to issues that keep the entire thing from working, but it's a freebie, so I get it I guess.

**Comment by Comet-boy** [comment]
I’ve just solved my sound issue by creating a “ asound.conf” file as per this article:

 

Setting the default device - AlsaProject (alsa-project.org) <https://www.alsa-project.org/wiki/Setting_the_default_device> 

 

Have you tried doing this??

 

Bruce

**Comment by zarmedia1** [comment]
Thanks Bruce.  My original request for help was about 2 years ago, so this project has been long shelved, but I'm interested in bringing it out again because I have some slot machines I want to convert to jukeboxes.  I will definitely refer back to this when that times comes, thanks a lot for sharing!

**Comment by GRVLK** [comment]
Try leaving out the 'sudo' on the 2nd line. Not sure why this works but it did for me.
